src/searchbar.h

```diff
@@ -46,11 +46,15 @@ public slots:
     bool m_returnPressed = false;
     QTimer* mp_typingTimer;
     int m_token;
+    bool m_scrolledEndBefore = false;
```

Comment:
1. Explain the meaning of this data member here
2. Consider renaming it to something better (e.g. `m_haveAlreadyScrolledToEnd`, `m_onTheVergeOfScrollingPastEnd`, `m_aboutToScrollPastEnd`, `m_justShortOfScrollingPastEnd`, etc)

Reply:
Done